project/1.18/assets/tinkers-construct/tconstruct/book/materials_and_you/zh_cn/intro/pattern_craft.json

```
@@ -3,6 +3,6 @@
   "recipe": "tconstruct:tables/pattern",
   "grid_size": "small",
   "description": [
-    { "text": "你将会需要许多模具来制作工具，制作工具部件和部件制造台的时候都用得到。" }
+    { "text": "你将会需要许多模具来制作工具，制作工具部件和部件制造台也是如此。" }
```

```suggestion
    { "text": "你将会需要许多模具来制作工具，制作工具部件和部件制造台的时候都用得到。" }
```
这里是“都用得到”还是“也用得到”？看一下原文，选择比较合适的那个。

---

原文：`You will need many patterns as you create tools, both to create tool parts and worktables.`